test/config/utils.js

```diff
@@ -36,7 +36,7 @@ const sharedTokenProperties = {
   token: {
     sessionId: "7b794d68-dc94-4829-bd68-596c5d2693b5",
     iat: new Date().getTime(),
-    exp: new Date().getTime(),
+    exp: new Date().getTime() + 86400000, // 1 day
```

Comment:
@tyrw These should be in seconds, not milliseconds. See `NumericDate` definition [here](https://www.rfc-editor.org/rfc/rfc7519#page-6).

Reply:
Thanks!